### DISCUSSION TITLE
Access to Google Sheet

### DISCUSSION THREAD

**Discussion by fifson1** [chriskyfung/AutoFetcher-IG-Stories-to-GDrive#48] [Q&A] [answered]
Hello, 
I can't copy new version Google Sheet, because I don't have access. Can you solve this?

**Comment by chriskyfung** [comment, marked as answer]
The URL has been corrected to be <https://docs.google.com/spreadsheets/d/1C3o6AkN2QzzITbKg_1Blzz8H9faVQQ6sHOzLxjWQ6P4/copy> with a filename of **[v4] IG-Stories-AutoFetcher (Updated on 2022-08-23)**

**Reply by fifson1** [reply]
Thank you, now I can copy. I want to ask: new version may help with problems with logging out from IG? I still have problems with this. After few days IG auto logged me from account and I still must copy new cookies...

**Reply by chriskyfung** [reply]
@fifson1 Please keep discussing the login issue within #45
